src/index.ts

```diff
@@ -77,7 +94,7 @@ export class UnleashClient extends TinyEmitter {
         this.clientKey = clientKey;
         this.storage = storageProvider || new LocalStorageProvider();
         this.refreshInterval = refreshInterval * 1000;
-        this.context = {environment, appName};
+        this.context = {environment, appName, userId, sessionId, remoteAddress, properties};
```

Comment:
Since userId, sessionId, ... could be undefined, should they be conditionally added to the context?

Reply:
I think the context can have these static keys, but they should be omitted when parsed into the query params?